tests_new/integration_tests/README.md

```diff
@@ -3,7 +3,18 @@ The purpose of these tests is to automatically validate functionalities of data.
 
 ## Pre-requisites
 - A real deployment of data.all in AWS
-- 4 Cognito users (at the moment only Cognito is supported) like the ones defined in `conftest`(e.g. `testUser1` with password `Pass1Word!`)
+- A SecretManager secret (`'{self.resource_prefix}-{target_env["envname"]}-cognito-test-users'`) with the followings contents
```

Comment:
Do we need SecretManager here? We don't need auto rotation or so, I think SSM Parameter store is enough, also it's free.

Reply:
That's a good point, I am not an expert but my go-to is that when a password is involved I use SecretsManager (even if it's just for tests). Apart from the pricing aspect I don't see a downside.

Reply:
We had issues in the past with checkov flagging some secrets in SecretsManager because some best practices dictate that encrypted Secrets in secret manager should be rotated at least annually and encrypted with a KMS key. So if we add a secret, it needs to be encrypted with KMS and with rotation enabled. So as @SofiaSazonova was saying if it does not rotate, we might go for SSM. The question is, do we need to rotate them? what can happen if the password gets exposed?

In the scenario where a malicious user:
- knows that a company uses data.all. Tries to guess the cloudfront domain that might be something like <company-name>.dataall.com
- tries to hit the domain with the default test user password (company is careless and did not change it)
- can overflood a company CICD pipeline with requests?

Reply:
After discussing with @dlpzx offline we decide to use SSM params for simplicity. Thanks for the spot on commend Sofia :)